chainstate/src/rpc/types/block.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2022 RBB S.r.l
```

Comment:
Come and join us in the future my man

Reply:
Nah... I like it here better. Now leave me alone 😄